src/shared/ui/Card/Card.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+
+import cls from "./Card.module.scss";
+
+export type CardProps = {
+  id?: number;
+  image: string;
+  category?: string;
+  title: React.ReactNode;
+  subtitle?: React.ReactNode;
+  content?: React.ReactNode;
+  onClick?: React.MouseEventHandler;
+};
+export const Card: FC<CardProps> = ({
+  id,
+  image,
+  category,
+  title,
+  subtitle,
+  content,
+  onClick,
+}) => {
+  return (
+    <div className={cls.card} onClick={onClick}>
+      <img className={cls.image} src={image} alt={""} />
+      <div className={cls.category}>{category}</div>
+      <div className={cls.title}>{title}</div>
+      <div className={cls.subtitle}>{subtitle}</div>
+      <div className={cls.content}>${content}</div>
+    </div>
+  );
+};
```

Comment:
тоже чисто чтобы красивее было, можно убрать лишние фигурные скобки и `return`:
```suggestion
}) => (
    <div className={cls.card} onClick={onClick}>
      <img className={cls.image} src={image} alt={""} />
      <div className={cls.category}>{category}</div>
      <div className={cls.title}>{title}</div>
      <div className={cls.subtitle}>{subtitle}</div>
      <div className={cls.content}>${content}</div>
    </div>
  )
```

некрит